packages/salesforcedx-vscode-core/src/commands/templates/forceApexUnitClassCreate.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { LocalComponent } from '@salesforce/salesforcedx-utils-vscode';
+import { CompositeParametersGatherer, MetadataTypeGatherer, OverwriteComponentPrompt, SfdxCommandlet, SfdxWorkspaceChecker, SimpleGatherer } from '../util';
+import { LibraryForceApexUnitClassCreateExecutor } from './executors/LibraryForceApexUnitClassCreateExecutor';
+import { getParamGatherers } from './forceApexClassCreate';
+import { APEX_CLASS_TYPE } from './metadataTypeConstants';
+
+export const forceApexUnitClassCreate = async (unitFileToCreate?: string, unitFileDirectory?: string) => {
```

Comment:
By taking optional parameters here I enable us to use this command for the E4D test case generation effort by using this core command.

Reply:
note I'm going to verify this works the way I expect (outside of unit tests) with the vsix generated in GHA.